ui/pages/swaps/select-quote-popover/index.scss

```
@@ -74,23 +74,23 @@
     cursor: pointer;
     color: var(--black);
     height: 49px;
-    border-bottom: 1px solid var(--Grey-100);
+    border-bottom: 1px solid var(--color-background-alternative);
     margin-bottom: 8px;
     border-radius: 8px;
     background: var(--Grey-000);
     padding-right: 12px;
     border: none;
 
     &:hover {
-      border: 1px solid var(--Blue-500);
+      border: 1px solid var(--text-primary-color);
```

text-primary-color doesn't exist but funnily enough it's closer to the design tokens v2
```suggestion
      border: 1px solid var(--color-primary-default);
```